batch/batch/driver/zone_monitor.py

```diff
@@ -117,6 +118,10 @@ async def update_region_quotas(self):
             for name in BATCH_GCP_REGIONS
         }
 
+        self.zones = [url_basename(z)
+                      for r in self.region_info.values()
+                      for z in r['zones']]
+
         log.info('updated region quotas')
```

Comment:
For a follow up PR: let's fix this terminology, this is no longer about quotas, it's about tracking the zones present in each region.